continuity.idpa.annotation/src/main/java/org/continuity/idpa/annotation/controllers/AnnotationController.java

```diff
@@ -134,6 +141,53 @@ public AnnotationController(AnnotationStorageManager storageManager) {
 			return new ResponseEntity<>(report.toString(), HttpStatus.CREATED);
 		}
 	}
+	
+	/**
+	 * Updates the annotation stored with the specified tag. If the annotation is invalid with
+	 * respect to the system model, it is rejected.
+	 *
+	 * @param tag
+	 * 				Tag of the annotation.
+	 * @param filePath
+	 * 				Path to the YAML annotation model.
+	 * @return
+	 * @throws FileNotFoundException Defined file not found.
+	 */
+	@RequestMapping(path = UPDATE, method = RequestMethod.PUT)
+	public ResponseEntity<String> updateAnnotation(@PathVariable("tag") String tag, 
```

Comment:
I don't get how this works. You pass a path to a file (which is e.g., on my local machine) and the server reads it. Does this work?!

Reply:
You are right. This works only if the server is on the local machine or if the file is on the server machine.

Reply:
Ok, that's not what I expected. There is already a possibility to upload an IDPA using the CLI - where the CLI can be stored on the local machine. Or you can upload it as JSON.
I don't see the benefit of this new endpoint.